libr/core/visual.c

```diff
@@ -1446,7 +1446,7 @@ R_API int r_core_visual_refs(RCore *core, bool xref, bool fcnInsteadOfAddr) {
 				} else {
 					RFlagItem *f = r_flag_get_at (core->flags, refi->addr, true);
 					if (f) {
-						name = r_str_newf ("%s + %d", f->name, refi->addr - f->offset);
+						name = r_str_newf ("%s + %" PFMT64d, f->name, refi->addr - f->offset);
```

Comment:
refi->addr should be always greather than f->offset... can you asser that? or just soft fix to avoid "x + -1" things

Reply:
It's out of the scope of this pull request.